### PR TITLE
Add documentation for IptablesAllowAction

### PR DESCRIPTION
**Pull request by Logan2211**
See https://github.com/projectcalico/felix/pull/1492

### CODE REVIEW

File: master/reference/felix/configuration.md [hunk 1]
```diff
@@ -118,4 +119,4 @@ to set a per-host override for "myhost" to move the log file to /tmp/felix.log:
 
 (For a global setting, omit the `--node=` option.)
 
-For more information, see the [calicoctl config documentation](../calicoctl/commands/config).
```
Comment (bcreane) [inline master/reference/felix/configuration.md]: nit: unintended new line inserted here?

Comment (Logan2211) [inline master/reference/felix/configuration.md]: Yeah it was unintentional. I can remove it if desired. My editor adds a line break at the end of the file if there is not one.

Comment (bcreane) [inline master/reference/felix/configuration.md]: If no one else care, I'm fine with the change since it's so minor.

Comment (caseydavenport) [inline master/reference/felix/configuration.md]: Yeah this is fine, no need to change :)

**Pull request merged**